src/Util.hpp

```diff
@@ -239,6 +239,16 @@ std::string real_path(const std::string& path,
 // extension as determined by `get_extension()`.
 nonstd::string_view remove_extension(nonstd::string_view path);
 
+// Split `input` into words at any of the characters listed in `separators`.
+// These words are a view into `input`, empty words are omitted.
+// `separators` must not be the empty string nor a nullptr.
+std::vector<nonstd::string_view> split_into_views(nonstd::string_view input,
+                                                  const char* separators);
+
+// Same as `split_into_views` but the words are copied from `input`.
```

Comment:
I'm convinced you already know how `std::string`s work, so I'm not sure I understand the question...?

Reply:
string_view is usually not null terminated. So I was wondering whether it's used correctly everywhere it is converted to char* via data(). I'll have a look during review.

Reply:
Oh, I thought you were talking about `split_into_strings` since you commented on its doc string.

No, the `string_views` produced by `split_into_views` do not refer to null-terminated data.